src/notation/qml/MuseScore/NotationScene/PercussionPanelPadSwapDialog.qml

```diff
@@ -0,0 +1,236 @@
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2024 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+
+import Muse.Ui 1.0
+import Muse.UiComponents 1.0
+
+StyledDialogView {
+    id: root
+
+    contentWidth: 500
+    contentHeight: contentRow.implicitHeight + contentColumn.spacing + buttonBox.implicitHeight
+
+    margins: 16
+
+    //! NOTE: Actually making this empty will display "MuseScore Studio"
+    title: " "
+
+    function done(data = {}) {
+        let value = Object.assign(data)
+
+        root.ret = {
+            errcode: 0,
+            value: value
+        }
+
+        root.hide()
+    }
+
+    onNavigationActivateRequested: {
+        var btn = buttonBox.firstFocusBtn
+        if (btn) {
+            btn.navigation.requestActive()
+        }
+    }
+
+    onAccessibilityActivateRequested: {
+        accessibleInfo.readInfo()
+    }
+
+    QtObject {
+        id: prv
+        property bool moveMidiNotesAndShortcuts: true
+    }
+
+    AccessibleItem {
+        id: accessibleInfo
+
+        accessibleParent: radioButtons.accessible
+        visualItem: titleLabel
+        role: MUAccessible.StaticText
+        name: titleLabel.text
+
+        function readInfo() {
+            accessibleInfo.ignored = false
+            accessibleInfo.focused = true
+        }
+
+        function resetFocus() {
+            accessibleInfo.ignored = true
+            accessibleInfo.focused = false
+        }
+    }
+
+    NavigationPanel {
+        id: contentNavigationPanel
+
+        name: "contentNavigationPanel"
+        section: root.navigationSection
+        order: 0
+    }
+
+    Row {
+        id: contentRow
+
+        width: parent.width
+        spacing: 28
+
+        StyledIconLabel {
+            id: icon
+
+            font.pixelSize: 48
+            iconCode: IconCode.QUESTION
+        }
+
+        Column {
+            id: contentColumn
+
+            width: contentRow.width - contentRow.spacing - icon.width
+            spacing: 16
+
+            StyledTextLabel {
+                id: titleLabel
+
+                width: parent.width
+
+                font: ui.theme.largeBodyBoldFont
+                horizontalAlignment: Text.AlignLeft
+                wrapMode: Text.Wrap
+
+                text: qsTrc("notation", "Do you also want to move the MIDI notes and keyboard shortcuts that trigger these sounds?")
```

Comment:
Perhaps we need a new context specifically for the percussion panel, like "notation/percussion"